plugins/foreman_openscap/0.4/index.md

```diff
@@ -0,0 +1,187 @@
+* smart_proxy_openscap - Smart-Proxy plugin which distributes SCAP content to hosts and post ARF reports from client hosts to Foreman
+* foreman_scap_client - A client script which runs OpenSCAP scan and uploads the scan report to the Smart-Proxy
+* puppet-foreman_scap_client - A puppet module which configures foreman_scap_client
+
+## 1.1 OpenSCAP Basic Concepts
+
+There are three basic concepts (entities) in OpenSCAP plug-in: SCAP Contents, Compliance Policies and ARF Reports.
+
+__SCAP Content__ represents SCAP DataStream XML file as defined by SCAP 1.2 standard. Datastream file contains implementation of compliance, configuration or security baselines. Users are advised to acquire examplary baseline by installing scap-security-guide package. DataStream file usualy contains multiple XCCDF Profiles. Each for different security target. The content of Datastream file can be inspected by oscap tool from openscap-scanner package.
+
+__Compliance Policy__ is highlevel concept of a baseline applied to the infrastructure. Compliance Policy is defined by user on web interface. User may assign following information to the Policy:
+
+* SCAP Content
+* XCCDF Profile from particular SCAP Content
+* Hostgroups that should comply with the policy
+* Schedule - the period in which the audit shall occur
+
+__ARF Report__ is XML output of single scan occurance per single host. Asset Reporting File format is defined by SCAP 1.2 standard. Foreman plug-in stores the ARF Reports in database for later inspections.
+
+
+## 1.2 Release Notes
+OpenSCAP has two versions: 0.3.4 for Foreman 1.7 and 0.4.x for 1.8+.
+
+Both versions' functionality is the same. The changes reflect the changes in communicating with Foreman API and Smart-Proxy.
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Foreman version</th>
+    <th>Plugin version</th>
+    <th>Proxy version</th>
+    <th>Client version</th>
+  </tr>
+  <tr>
+    <td>= 1.7</td>
+    <td>0.3.4</td>
+    <td>0.3.1</td>
+    <td>0.1.1</td>
+  </tr>
+  <tr>
+    <td>>= 1.8</td>
+    <td>0.4.1</td>
+    <td>0.4.0</td>
+    <td>0.1.1</td>
+  </tr>
+</table>
+
+# 2. Installation
+
+There are a few components to install: 
+
+Foreman OpenSCAP plugin, Foreman Proxy OpenSCAP plugin (aka Smart Proxy OpenSCAP), foreman_scap_client and puppet-foreman_scap_client.
+
+### 2.1 Installing foreman_openscap
+
+Please refer to Foreman install plugin [manual](http://theforeman.org/manuals/1.8/index.html#6.1InstallaPlugin) to install the plugin on Foreman.
+
+### 2.2 Installing smart_proxy_openscap
+
+On the Smart-Proxy server:
+
+    
+    > gem install smart_proxy_openscap -v '<version>'
+    > cp /path/to/gem/smart_proxy_openscap-<version>/settings.d/openscap.yml.example /path/to/foreman-proxy/settings.d/openscap.yml
+    
+Edit ```openscap.yml``` with the appropriate settings
+
+    ---
+    :enabled: true
+
+
+### 2.3 Installing puppet-foreman_scap_client
+
```

Comment:
I think this deserves a bit more explanation. The module needs to be installed into puppet environment to which client hosts will be assigned.